converter/datamodel.py

```diff
@@ -227,6 +225,10 @@ def __init__(self, alias, accession, description, protocol_type, hardware, softw
         self.software = software
         self.parameters = parameters
 
+    def __repr__(self):
```

Comment:
This seems to be some introspection? I would suggest adding a couple of lines of comment to explain what you are after here.

Reply:
I've added this to a few of the classes mainly for debugging. The __repr__ is a special Python function that gives the object a human readable output (to get an idea of the content of the object) when you run print() or str() on it.